test/script/run_tests.sh

```diff
@@ -53,6 +53,8 @@ run_test_suite () {
   else
     nyc node test/test.js
   fi
+
+  npm run test-types
```

Comment:
Not sure if it's good to run this after the main tests 🤔 On the other hand it would probably give weird errors if say there were a bug in the agent code somewhere and we triggered that bug by accident from inside the type tests if they ran before the regular agent tests. So I'd prefer if we could run them after the main tests.

@kuisathaverat do you think Jenkins would be ok with running this command after the `JUNIT` block above?

Reply:
if this command does not delete or overwrite the result files from tests and coverage, it'd be ok

Reply:
It shouldn't. So I guess that's ok then. Thanks 😃